app/main.py

```diff
@@ -16,4 +16,20 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+variables = [lucky_number,
+             pi,
+             one_is_a_prime_number,
+             name,
+             my_favourite_films,
+             profile_info,
+             marks,
+             collection_of_coins]
+sorted_variables = {"mutable": [], "immutable": []}
```

Comment:
you should put it manually all variables
sorted_variables = {"mutable": [here], "immutable": [here]}

Reply:
I can not catch what do you mean. Can you explain, please?

Reply:
<img width="371" alt="image" src="https://user-images.githubusercontent.com/72562232/198220363-d181e02e-b447-4171-8581-21ea358e7bb8.png">
you don't need loops, and other tools, just make a variable